src/main/java/org/opentripplanner/openstreetmap/model/OSMNode.java

```diff
@@ -38,10 +38,14 @@ public boolean isMultiLevel() {
     return hasTag("highway") && "elevator".equals(getTag("highway"));
   }
 
-  public boolean hasTrafficLight() {
+  public boolean hasHighwayTrafficLight() {
```

Comment:
Can these be renamed into what they are used for? Are they for vehicles and pedestrians respectively?

Reply:
Highway traffic light is also used for cyclists on car streets, I thought about this too and I ended up naming these like this so how these are used in different types of routing is decided elsewhere.